Warning/Error when building EVS UT for RTEMS/LEON3

**Describe the bug**
Warning when building EVS UT for RTEMS/LEON3 causes an error if building with -Wall.

```
/home/jfmarti4/remotes/ds_cfs/cfe/modules/evs/ut-coverage/evs_UT.c:328:14: error: cast increases required alignment of target type [-Werror=cast-align]
  328 |     bufPtr = (CFE_SB_Buffer_t *)&testMsg; /* Fake Test Message */
      |              ^
cc1: all warnings being treated as errors
```

**To Reproduce**
Steps to reproduce the behavior:
1. Build with sparc-gaisler-rtems5-gcc with the -Wall flag

**Expected behavior**
Successful build.

**Code snips**
If applicable, add references to the software.

**System observed on:**
 - Building on Ubuntu 20.04 server

**Additional context**
Add any other context about the problem here.

**Reporter Info**
Jose F. Martinez Pedraza | GSFC 582

--- a/modules/evs/ut-coverage/evs_UT.c
+++ b/modules/evs/ut-coverage/evs_UT.c
@@ -293,10 +293,15 @@ void Test_Init(void)
     CFE_EVS_EnableAppEventTypeCmd_t appbitcmd;
 
     CFE_SB_Buffer_t *bufPtr;
-    CFE_EVS_NoopCmd_t testMsg;
-
+    union 
+    {
+        CFE_SB_Buffer_t   SbBuf;
+        CFE_EVS_NoopCmd_t Noop;
+    } TestMsg;
+    
     UtPrintf("Begin Test Init");
 
+    memset(&TestMsg, 0, sizeof(TestMsg));
     memset(&bitmaskcmd, 0, sizeof(bitmaskcmd));
     memset(&appbitcmd, 0, sizeof(appbitcmd));
 
@@ -325,7 +330,7 @@ void Test_Init(void)
     /* Set unexpected message ID */
     UT_SetupBasicMsgDispatch(&UT_TPID_CFE_EVS_INVALID_MID, 0, true);
 
-    bufPtr = (CFE_SB_Buffer_t *)&testMsg; /* Fake Test Message */
+    bufPtr = &TestMsg.SbBuf; /* Fake Test Message */
     UT_SetDataBuffer(UT_KEY(CFE_SB_ReceiveBuffer), &bufPtr, sizeof(bufPtr), false);
 
     UT_EVS_DoGenericCheckEvents(CFE_EVS_TaskMain, &UT_EVS_EventBuf);

